pkg/apis/v1alpha2/crd_validation_cel_test.go

```diff
@@ -59,6 +59,42 @@ var _ = Describe("CEL/Validation", func() {
 			},
 		}
 	})
+	Context("VnetSubnetID", func() {
+		DescribeTable("Should only accept valid VnetSubnetID", func(vnetSubnetID string, expected bool) {
```

Comment:
nit:
```suggestion
		DescribeTable("Should only accept valid VnetSubnetID", func(vnetSubnetID string, expectValid bool) {
```

Or some other similar naming. Just `expected` is minorly unclear.